hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/RawLocalFileSystem.java

```diff
@@ -267,8 +282,84 @@ public boolean hasCapability(String capability) {
     public IOStatistics getIOStatistics() {
       return ioStatistics;
     }
+
+    // QQ: Should we make this synchronized?
```

Comment:
yes. Also needs to reject if file is closed.

Reply:
isClosed is not implemented in RawLocalFileSystem. This needs to be added separately.